src/tests/dbus-tests/test_nvme.py

```diff
@@ -229,6 +231,14 @@ def tearDownClass(cls):
                 pass
         udiskstestcase.UdisksTestCase.tearDownClass()
 
+    def _get_objects(self):
+        objects = safe_dbus.call_sync(self.iface_prefix,
```

Comment:
Do we need to use `safe_dbus` here? We use this on other places to get all objects:

```
        udisks = self.get_object('')
        objects = udisks.GetManagedObjects(dbus_interface='org.freedesktop.DBus.ObjectManager')
```